docs/_assets/style.css

```diff
@@ -1,23 +1,62 @@
 body[layout='layout-home'] .markdown-body .call-to-action:nth-of-type(2) {
-  --primary-color: #222;
-  --primary-color-lighter: #333;
+  --primary-color: #333;
+  --primary-color-lighter: #444;
   --primary-color-darker: #000;
 }
 
+body[layout='layout-home'] .markdown-body .call-to-action:nth-of-type(1) {
+  --primary-color: #8f7325;
+  --primary-color-lighter: #8f7325;
```

Comment:
As far as i can find the `--primary-color-lighter` is only used in call-to-actions. Don't override the value here, but in variables.css.